protocol/communities/manager_test.go

```diff
@@ -45,28 +44,38 @@ func TestManagerSuite(t *testing.T) {
 
 type ManagerSuite struct {
 	suite.Suite
-	manager *Manager
+	manager        *Manager
+	torrentManager TorrentContract
```

Comment:
hmm, `torrentManager` is casted to `TorrentManager` many times. 
Maybe just cast it during `SetupTest`?

Reply:
Yeah, that is a bit of a mess. I'll change the `ManagerSuite` declaration to explicitly be `*TorrentManager` (or `ArchiveManager` after I've renamed it) and then remove all type casting.

Reply:
https://github.com/status-im/status-go/pull/5257/commits/1b8a05cd1b2018285a99202baafbfa7887b2f540